content/02.main-text.md

```diff
@@ -20,8 +20,54 @@ Although we requested that some authors participate for their specific expertise
+All citations point to a standard identifier, for which Manubot automatically retrieves bibliographic metadata.
+Currently, citations to DOIs (Digital Object Identifiers), PubMed identifiers, arXiv identifiers, and URLs (web addresses) are supported.
+Metadata is retrieved using DOI [Content Negotiation](https://citation.crosscite.org/docs.html), NCBI's [Citation Exporter](https://www.ncbi.nlm.nih.gov/pmc/tools/ctxp/), the [arXiv API](https://arxiv.org/help/api/index), or [Greycite](http://greycite.knowledgeblog.org/) [@arxiv:1304.7151v1].
+Metadata is exported to [CSL JSON Items](http://citeproc-js.readthedocs.io/en/latest/csl-json/markup.html#items), an open standard that's widely supported by reference managers [@doi:10.1007/978-3-319-00026-8_8; @doi:10.1080/02763869.2012.641841].
+In cases where automatic retrieval of metadata fails or produces incorrect references -- which is most common for URL citations -- users can manually provide the correct CSL JSON.
+
+The Manubot formats bibliographies according to a [Citation Style Language](http://citationstyles.org/) (CSL) specification.
+As a result, users can choose from thousands of existing CSL styles or use Manubot's default style.
+Styles define how references are constructed from bibliographic metadata, controlling layout details such as the max number of authors to list per reference.
+Thousands of journals have [predefined styles](http://editor.citationstyles.org/searchByName/).
+As a result, adopting the specific bibliographic format required by a journal usually just requires specifying the style's source URL in the Manubot's configuration.
+
+Manubot uses [Pandoc](https://pandoc.org/) to convert manuscripts from markdown source to HTML (and optionally DOCX) output.
+Pandoc supports conversion between a large number of formats, offering Manubot users broad interoperability and preventing platform lock-in.
+For example, Manubot intentionally does not rely on LaTeX, although users are free to export manuscripts to LaTeX themselves.
+In a future release, Pandoc [will support](https://github.com/greenelab/manubot-rootstock/pull/51#issuecomment-321949622) export to [Journal Article Tag Suite](https://jats.nlm.nih.gov/) (JATS).
+JATS is a standard XML format for scholarly articles that is used by publishers, archives, and text miners [@url:http://www.niso.org/standards/z39-96-2015/; @doi:10.6087/kcse.2014.1.99; @doi:10.1080/00987913.2012.10765464].
+For now however, the primary Manubot output is HTML intended to be viewed in a web browser.
+From the rendered HTML, Manubot also exports to PDF.
+
+Manubot performs continuous publication: every update to a manuscript's source is automatically reflected in the online outputs.
+The approach uses continuous integration (CI) [@doi:10.1038/550143a; @doi:10.1038/nbt.3780], specifically [Travis CI](https://travis-ci.org/) at the moment, to monitor for changes.
+When changes occur, the CI service rebuilds the manuscript.
```

Comment:
Is `generates updated HTML and PDF documents` more understandable than `rebuilds the manuscript`?